### DISCUSSION TITLE
Welcome to k8s-staticroute-operator Discussions!

### DISCUSSION THREAD

**Discussion by bikramgupta** [digitalocean/k8s-staticroute-operator#17] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by priyankashiv2513** [comment]
root@pool-8-8-basic-aayup:~# curl ifconfig.me/ip -v
*   Trying 34.160.111.145:80...
*   Trying [2600:1901:0:b2bd::]:80...
* Immediate connect fail for 2600:1901:0:b2bd::: Network is unreachable. i am not able to use gateway. route is there in node root@pool-8-8-basic-aayup:~# route -n
Kernel IP routing table
Destination     Gateway         Genmask         Flags Metric Ref    Use Iface
0.0.0.0         128.199.16.1    0.0.0.0         UG    0      0        0 eth0
10.0.0.0        10.162.0.1      255.0.0.0       UG    101    0        0 eth1
10.47.0.0       0.0.0.0         255.255.0.0     U     0      0        0 eth0
10.108.5.128    10.162.0.13     255.255.255.128 UG    0      0        0 eth1
10.108.6.0      10.162.0.10     255.255.255.128 UG    0      0        0 eth1
10.108.6.128    10.108.6.134    255.255.255.128 UG    0      0        0 cilium_host
10.108.6.134    0.0.0.0         255.255.255.255 UH    0      0        0 cilium_host
10.162.0.0      0.0.0.0         255.255.0.0     U     0      0        0 eth1
34.160.111.145  10.162.0.2      255.255.255.255 UGH   0      0        0 eth1
128.199.16.0    0.0.0.0         255.255.240.0   U     0      0        0 eth0
172.16.0.0      10.162.0.1      255.240.0.0     UG    101    0        0 eth1
192.168.0.0     10.162.0.1      255.255.0.0     UG    101    0        0 eth1

**Comment by bikramgupta** [comment]
If you can wait for 1-2 weeks, we are in the process of shipping a managed static route operator. It will support

- 0.0.0.0 routes
- multiple destinations as gateways with ECMP routing

If you need it urgently, just reply so and I will try to share the image/instructions on Mon/Tue.
